MiniApp/Classes/ui/MiniAppViewController.swift

```diff
@@ -32,6 +32,16 @@ public class MiniAppViewController: UIViewController {
         didSet { update() }
     }
 
+    public override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        NotificationCenter.default.sendCustomEvent(MiniAppEvent.Event(type: .resume, comment: "MiniApp view did appear"))
+    }
+
+    public override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        NotificationCenter.default.sendCustomEvent(MiniAppEvent.Event(type: .resume, comment: "MiniApp view will disappear"))
```

Comment:
This is supposed to a `.pause` event right ??

Reply:
nice catch